QUEST.tsv

```diff
@@ -2664,34 +2664,34 @@ QUEST_20151001_002671
 QUEST_20151001_002679	You have to pay your life!
 QUEST_20151001_002680	Truth Token
 QUEST_20151001_002681	The divine energy decreased when the truth token was lighting.
-QUEST_20151001_002682	The mysterious girn shown up with light. Let's talk.
-QUEST_20151001_002683	The mysterious girl run without any comments. Follow her.
+QUEST_20151001_002682	The mysterious girl with the light has appeared. Talk to her.
+QUEST_20151001_002683	The mysterious girl ran away without saying anything. Follow her.
 QUEST_20151001_002684	Suspicous Basket
-QUEST_20151001_002685	An offensive smell greeted my nose. It seems old foods.
-QUEST_20151001_002686	The mysterious girl shown with the light again.
-QUEST_20151001_002687	Welcome! There is no time!{nl} My strength is to here!
+QUEST_20151001_002685	The offensive smell is unpleasant to your nose. It seems to be old food after it was hidden.
+QUEST_20151001_002686	The mysterious girl with the light has appeared once again.
+QUEST_20151001_002687	Welcome! There is no time!{nl} My strength is over here!
 QUEST_20151001_002688	Aldona! Giderone!{nl}Now!
-QUEST_20151001_002689	Aldona and Giderone..{nl}I'm sorry for burdened you due to the feebleness.{nl}
-QUEST_20151001_002690	But time is not enoug.{nl}You guys brought the Revelator.. He has a power to remove the disaster at here.{nl}
-QUEST_20151001_002691	I will ask a revelator to help me.{nl}You guys shoud to ready to make the key of evening sky quickly.{nl}
+QUEST_20151001_002689	Aldona and Giderone..{nl}I'm sorry for being a burden to you because of my feebleness.{nl}
+QUEST_20151001_002690	However, let's not waste any time.{nl}You brought the Savior.. who has the power to get rid of this disaster here.{nl}
+QUEST_20151001_002691	I will ask the Savior to help me.{nl}You guys should be ready to quickly make the Evening Star Key.{nl}
 QUEST_20151001_002692	The many deaths of demons could not fill the emptiness in my heart. {nl}Once the Goddess's great mission for me is over, I will be left with a long life full of boring moments.
-QUEST_20151001_002693	Damn, I thought it would die after getting its throat cut.{nl}I hope it can just behave and die slowly now.
-QUEST_20151001_002694	Lie revelation..you want to lure me just using it?{nl}It's so worried the preparation what Laima was prepared at best.
-QUEST_20151001_002695	But, you can't do anything for me with the your preparation.{nl}I smile since your foolish that the gap of strengthen absoultely.
+QUEST_20151001_002693	Ugh, I thought it would die after getting its throat cut.{nl}I hope it can just behave now and die slowly.
+QUEST_20151001_002694	A false revelation.. Are you using it to lure me?{nl}It's so worried the preparation what Laima was prepared at best.
+QUEST_20151001_002695	But, you can't do anything for me with your preparation.{nl}I'm smiling because of your foolishness and your gap of absolute power.
```

Comment:
"But, you can't do anything against me with your preparation."
"I'm smiling because of your foolishness and the gap between your power and mine." (?)